src/app/collaborators/page.tsx

```
@@ -79,144 +142,333 @@ export default function CollaboratorsPage() {
+                        Be part of a growing community dedicated to improving transit information
+                      </p>
+                    </div>
+                  </div>
+                </div>
               </div>
             </div>
+
+            {/* Back to top button */}
+            <div className="flex justify-center mt-12">
+              <a 
+                href="#" 
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)